va/http.go

```diff
@@ -397,12 +347,39 @@ func (va *ValidationAuthorityImpl) processHTTPValidation(
 		return nil, nil, err
 	}
 
+	// Create an initial GET Request
+	initialURL := fmt.Sprintf("http://%s%s", host, path)
+	initialReq, err := http.NewRequest("GET", initialURL, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	// Immediately reconstruct the request using the validation context
+	initialReq = initialReq.WithContext(ctx)
+	if va.userAgent != "" {
+		initialReq.Header.Set("User-Agent", va.userAgent)
+	}
+	// Some of our users use mod_security. Mod_security sees a lack of Accept
+	// headers as bot behavior and rejects requests. While this is a bug in
+	// mod_security's rules (given that the HTTP specs disagree with that
+	// requirement), we add the Accept header now in order to fix our
+	// mod_security users' mysterious breakages. See
+	// <https://github.com/SpiderLabs/owasp-modsecurity-crs/issues/265> and
+	// <https://github.com/letsencrypt/boulder/issues/1019>. This was done
+	// because it's a one-line fix with no downside. We're not likely to want to
+	// do many more things to satisfy misunderstandings around HTTP.
+	initialReq.Header.Set("Accept", "*/*")
+
 	// Set up the initial validation request and a base validation record
-	initialReq, baseRecord, err := va.setupHTTPValidation(ctx, nil, target)
+	dialer, baseRecord, err := va.setupHTTPValidation(ctx, initialReq.URL.String(), target)
+	fmt.Printf("Setup err: %s %#v\n", err, err)
```

Comment:
Debug printf.
```suggestion
```

Reply:
Thanks for catching that :blush:  Fixed in 82dba4d